GetStream/stream-chat-android
# [i196] support auto-translation feature in Compose

### 🎯 Goal

Related ticket: https://github.com/GetStream/android-internal-board/issues/196

### 🎨 UI Changes

#### Compose

| Before | After |
| --- | --- |
| ![Before](https://github.com/GetStream/stream-chat-android/assets/1286516/7dfb1a70-f13e-4572-912a-978c190e64d4) | ![After](https://github.com/GetStream/stream-chat-android/assets/1286516/ecd67b45-c086-43ff-b093-c3503ea817fc) | 

### 🧪 Testing

<details>

<summary>Patch that modifies <b>Compose Sample</b> app</summary>

```
Subject: [PATCH] Compose Sample
---
Index: stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/data/PredefinedUserCredentials.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/data/PredefinedUserCredentials.kt b/stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/data/PredefinedUserCredentials.kt
--- a/stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/data/PredefinedUserCredentials.kt	(revision ec17b6f91cc4fd2aba271cf88f42e6ccc3e28110)
+++ b/stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/data/PredefinedUserCredentials.kt	(date 1698105484400)
@@ -23,10 +23,20 @@
  */
 object PredefinedUserCredentials {
 
-    const val API_KEY: String = "qx5us2v6xvmh"
+    const val API_KEY: String = "2yxg5ueum4b7"
 
     val availableUsers: List<UserCredentials> = listOf(
         UserCredentials(
+            apiKey = API_KEY,
+            user = User().apply {
+                id = "codysmith"
+                name = "Cody Smith"
+                image = "https://ca.slack-edge.com/T02RM6X6B-U01PXET9GBH-6284119883ed-512"
+                language = "fr"
+            },
+            token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiY29keXNtaXRoIn0.2KhAQMOpybbTkj96mZqts3b4y7oSkE-QNx1L2LmzgPo"
+        ),
+        /*UserCredentials(
             apiKey = API_KEY,
             user = User().apply {
                 id = "jc"
@@ -169,6 +179,6 @@
                 image = "https://getstream.imgix.net/images/random_svg/QT.png"
             },
             token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoicWF0ZXN0MiJ9.GYp9ikLtU2eG9Mq7tmHThzbV7C8W82j18sExuO7-ogc"
-        ),
+        ),*/
     )
 }

```

</details>

#### Test Steps:

1. Apply the patch above
2. Login as `Cody`
3. Open `ViyaneBastos, ,` channel
4. Make sure you can see `Translated to French`

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
